api/start_server.sh

```diff
@@ -10,13 +9,11 @@ fi
 chmod -R 777 db
 chown -R www-data:www-data db
 python ./manage.py collectstatic
-mkdir /home/staticfiles
+mkdir -p /home/staticfiles
 mv static_root/* /home/staticfiles
 rm -rf static_root/
-(/usr/bin/redis-server) &
-(gunicorn app.wsgi --user www-data --bind 0.0.0.0:8000 --workers 3 --timeout 300) &
 (celery -A app worker --concurrency=2 -l INFO --purge) &
 (celery -A app worker --concurrency=4 -Q anomalySubTask -l INFO --purge) &
 (celery -A app beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler) &
-echo "starting nginx"
-nginx -g "daemon off;"
```

Comment:
This file is being used in current deployment. Don't update it unless deployment for production is updated.

Reply:
I have I have removed this file's changes.